src/App.tsx

```diff
@@ -1,61 +1,34 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo, TodoWithUser } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+import { FormAddTodo } from './components/FormAddTodo/FormAddTodo';
+import { findUserById } from './utils/findUserById';
+
+const preparedTodos = (todos: Todo[]) => {
+  return todos.map((todo: Todo) => ({
+    ...todo,
+    user: findUserById(usersFromServer, todo.userId),
+  }));
+};
```

Comment:
Please consider @Moroz-Dmytro comment
At the very least you need to rename this function, as current name is confusing